elisp-bytecode.texi

```diff
@@ -1063,6 +1132,7 @@ function for them.
 * concat2::
 * concat3::
 * concat4::
+* concatN::
```

Comment:
I had deliberately excluded concatN and listN from this section, but I think it's better to document them here rather than having a special section for them, now.